### ISSUE TITLE
Empty tag field is not included as param

### ISSUE THREAD

**Issue by hsyn** [aehlke/tag-it#169]
I realised if you remove all the tags for the field or post the form without adding any tag, there is no parameter sent to back-end saying that this field is empty. Because it is not sent at all.
An example:
Front-end looks like-->
Name: "yada yada"
Hobbies: (no tags)

Back-end recieves-->
params:   "name" --> "yada yada"

It would be better if it includes taggable field with empty value
params:   "name" --> "yada yada", "hobbies" --> ""

So, basically if no tag is present, generate an input field with empty value for the field name.
